spec/lib/govuk_proxy/static_proxy_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+require "govuk_app_config/govuk_proxy/static_proxy"
+require "rack/mock"
+
+RSpec.describe GovukProxy::StaticProxy do
+  # dummy app to validate success
+  let(:app) { ->(_env) { [200, {}, "success"] } }
+  let(:app_domain) { "app.domain" }
+  let(:static_domain) { "static.domain" }
+
+  subject { GovukProxy::StaticProxy.new(app, backend: "https://static.domain", streaming: false) }
```

Comment:
I think this use of [unnamed subject](https://rspec.rubystyle.guide/#use-subject) would fall foul of RSpec linting rules. I've had a look and they're not on for this app so predictably running them is a bit grim. May want to just change this to 

`let(:proxy) { described_class.new(app, backend: "https://static.domain", streaming: false) }` to not leave linting issues.

Reply:
Updated - thanks!